src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/integrationtest/TransformationBulkScanTest.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.integrationtest;
+
+import org.hamcrest.Matcher;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.reform.authorisation.validators.AuthTokenValidator;
+import uk.gov.hmcts.reform.bsp.common.model.validation.out.OcrValidationResult;
+import uk.gov.hmcts.reform.finrem.caseorchestration.CaseOrchestrationApplication;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.validation.FormAValidator;
+
+import java.util.Arrays;
+
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.isJson;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.SERVICE_AUTHORISATION_HEADER;
+
+@ContextConfiguration(classes = CaseOrchestrationApplication.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@PropertySource(value = "classpath:application.yml")
+@AutoConfigureMockMvc
+@RunWith(SpringRunner.class)
+public class TransformationBulkScanTest {
+    private static final String TRANSFORMATION_URL = "/transform-exception-record";
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    AuthTokenValidator authTokenValidator;
+
+    @MockBean
+    FormAValidator bulkScanFormValidator;
+
+    @Before
+    public void setup() {
+        when(authTokenValidator.getServiceName(any())).thenReturn("bulk_scan_orchestrator");
+    }
+
+    @Test
+    public void shouldReturnErrorResponseForInvalidData() throws Exception {
+        String jsonPayload = "{\"form_type\": \"formA\",\n" +
```

Comment:
This should be moved to a json file and loaded as it's done here:
https://github.com/hmcts/div-case-orchestration-service/pull/735/files#diff-84c37f3431f8ccdbcf88d5eea4167aedR278

Reply:
done